crud-s3/src/create_or_update.py

```diff
@@ -0,0 +1,16 @@
+import boto3
+import json
+import os
+
+def handler(body, req):
+    session = boto3.Session(
+        aws_access_key_id=os.environ['AWS_ACCESS_KEY'],
+        aws_secret_access_key=os.environ['AWS_SECRET_KEY'],
+    )
+    bucket = os.environ['S3_BUCKET_NAME']
```

Comment:
style is inconsistent. please use pep8 or something similar to check it

Reply:
Adding pycodestyle (new rebranding of pep8) and will add to readme